python/src/trezorlib/cli/cardano.py

```diff
@@ -69,7 +69,7 @@ def sign_tx(
     transaction = json.load(file)
 
     if testnet:
-        protocol_magic = cardano.PROTOCOL_MAGICS["testnet"]
+        protocol_magic = cardano.PROTOCOL_MAGICS["testnet_legacy"]
```

Comment:
Perhaps we could set the default to `cardano.PROTOCOL_MAGICS["testnet_preprod"]` as that should be the "new" testnet AFAIK? Won't `testnet_legacy` be shutdown at some point?

Same goes for `get_address`.

Reply:
Yes that makes sense. On the other hand, it might break some user scripts, but maybe that's negligible?

Reply:
As discussed offline, resolved by adding the testnet options.